Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -75,13 +79,25 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
+        $fixturesPath = DIRECTORY_SEPARATOR . 'DataFixtures' . DIRECTORY_SEPARATOR . 'ORM';
         $dirOrFile = $input->getOption('fixtures');
         if ($dirOrFile) {
-            $paths = is_array($dirOrFile) ? $dirOrFile : array($dirOrFile);
+
+            $paths = array();
+            $pathsParams = is_array($dirOrFile) ? $dirOrFile : array($dirOrFile);
+            foreach ($pathsParams as $path) {
+                if (preg_match('/^\@([^\/\\\\]+)(.*)$/', $path, $matches)) {
+                    $bundle = $this->getApplication()->getKernel()->getBundle($matches[1]);
```

Comment:
Out of scope of responsibility for this code. This should be handled by Kernel. But will add a testcase which mimic this behaviour.